environment-py3.yml

```
@@ -80,6 +80,10 @@ dependencies:
   #- tornado >=5.0.0,<6.0.0
   - typing >=3.6.6
   - pyyaml
+  # OAuth2
+  - authlib <=0.15.3
+  - pyjwt <=1.7.1
```

Ignoring for a moment the comment in another thread about py3 only, shouldn't this be
```suggestion
  - authlib >=0.15.3
  - pyjwt >=1.7.1
```
?

---

The code is brought to work with these versions. These versions of packages should support python3 also.

---

fixed to
```
authlib >=1.0.0
pyjwt >=2.1.0
```